src/views/CreateView.vue

```diff
@@ -343,6 +347,17 @@ export default {
         this.isFormSubmitted = true
       }
     }
+  },
+  mounted() {
+    const fetchExercises = async () => {
+      const { data } = await supabase
+        .from('Exercises')
+        .select()
+
+      this.allExercises = data
```

Comment:
```js
const response = await supabase.from('Exercises').select()

if (response.ok) {
  this.allExercises = response.data;
} else {
  // show toast with error msg
}
```

Reply:
**Corrected**, but could you please explain me what's the difference between:
```js
const { data } = await supabase
        .from('Exercises')
        .select()
```

and

```js
const response = await supabase.from('Exercises').select()
```

Also, I couldn't use `response.ok` because this response object didn't have such a property. So I changed it for `response.error === null`. Maybe I could do just `!response.error` but I seem to no longer know when I can use this syntax.